typescript/infra/src/config/funding/alert-query-templates.ts

```diff
@@ -32,7 +40,40 @@ export const LOW_URGENCY_KEY_FUNDER_FOOTER = `    # Mainnets that don't use key-
     (last_over_time(hyperlane_wallet_balance{wallet_name="relayer", chain="stride"}[1d])) - 10
 `;
 
-// TODO: add footer for LOW_URGENCY_ENG_KEY_FUNDER
+export const LOW_URGENCY_ENG_KEY_FUNDER_FOOTER = `    # Mainnets that don't use key-funder and all funds are stored in the relayer's wallet
+    # Eclipse
+    last_over_time(hyperlane_wallet_balance{wallet_name="relayer", hyperlane_context="hyperlane", chain=~"eclipsemainnet"}[1d]) - 0.5 or
```

Comment:
we have the ability to have these relayer balance alerts be dynamic using the `dailyRelayerBurn.json` / `desiredRelayerBalances.json` right? anyways I assume this can / should come later